content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -9,13 +9,15 @@ weight: 65
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.30" state="alpha" >}}
```

Comment:
I'm not sure what to document here: the first time this became available (1.26), the current status, or both?

If we include the current status, we have to refresh the version on each release. This had not been done for 1.28 and 1.29 because there were no other changes.

I had bumped the version from 1.26 to 1.27 because of the API change, but I am no longer sure whether that was the right thing to do, so I am going back to 1.26 here.

Reply:
We show readers whether the feature is alpha, beta, GA (or deprecated), and the version when the last transition occurred.

Provided you're targeting a recent-enough _dev-1.30_, you can do this:
```markdown
{{< feature-state feature_gate_name="DynamicResourceAllocation" >}}
```

and make sure that the feature gate file [`content/en/docs/reference/command-line-tools-reference/feature-gates/dynamic-resource-allocation.md`](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates/dynamic-resource-allocation.md) is current (I don't think there'd be any change needed).